web/src/views/DandisetLandingView/Meditor.vue

```diff
@@ -196,6 +193,10 @@ export default defineComponent({
       type: Object as PropType<DandiModel>,
       required: true,
     },
+    readonly: {
+      type: Boolean,
+      required: true,
```

Comment:
A default to `true` here might be nice

Reply:
No, you'd remove `required` and just have a default of `true`. Up to you but to me this prop reads like something that would have a default setting.

Reply:
See 9681a44adad9ad2d4c0547a9250727552e1722b5.